### ISSUE TITLE
how do you enable i2c interface for GPIO?

### ISSUE THREAD

**Issue by jata1** [ayufan-rock64/linux-build#199]
Hi, I have latest OMV / stretch (armhf) setup on my rock64. Working great. Thanks ayufan and other contributors.

I'd like to install a RTC on my board. It attaches to the GPIO and uses i2c. 

Is this possible? Any suggestions appreciated.

**Comment by ayufan** [comment]
Yes, just install. It is already supported.

**Comment by jata1** [comment]
That's great. I will try it and see if I can get it working.

**Comment by jata1** [comment]
I don't understand. It looks like there is a RTC already on my rock64 board - I haven't installed the RTC module.

May 31 09:03:04 rock64 kernel: [    2.263456] rk808-rtc rk808-rtc: rtc core: registered rk808-rtc as rtc0

and 
May 31 09:03:04 rock64 kernel: [    3.225131] rk808-rtc rk808-rtc: setting system clock to 2016-01-21 08:50:11 UTC (1453366211)

I think this is telling me there is a RTC embedded on the rock64 board. Is this possible?

**Comment by s-kostyuk** [comment]
@jata1, yes, it's possible. Rock64 uses RK805 power management chip with built-in RTC functionality. But, unfortunately, I failed to find a pin that can be used to power RTC while the rest of the board is not powered in the datasheet
